src/error.ts

```diff
@@ -0,0 +1,35 @@
+import chalk from 'chalk';
+
+export function printError(error: any) {
+  let s = '';
+
+  // skyerr
+  if (error && error.json && error.json.error) {
+    const skyerr = error.json.error;
+    switch (skyerr.code) {
+      // invalid argument
+      case 108:
```

Comment:
I think we should have an enum (or some constant declarations if you prefer) for possible error codes.

Reply:
Actually the full list of codes is defined in the skygear SDK. Should we duplicate that list here? Or we just add the code as we need it? Anyway I agree that we should introduce named constant for error code.